diff -Naurp openstack-common-0.1+git20111130.orig/run_tests.sh openstack-common-0.1+git20111130/run_tests.sh
--- openstack-common-0.1+git20111130.orig/run_tests.sh	2011-11-30 13:39:09.000000000 -0500
+++ openstack-common-0.1+git20111130/run_tests.sh	2011-12-05 13:07:02.913527709 -0500
@@ -24,7 +24,7 @@ function process_option {
     -N|--no-virtual-env) let always_venv=0; let never_venv=1;;
     -p|--pep8) let just_pep8=1;;
     -f|--force) let force=1;;
-    --unittests-only) noseargs="$noseargs --exclude-dir=tests/functional";;
+    --unittests-only) noseargs="$noseargs";;
     *) noseargs="$noseargs $1"
   esac
 }
diff -Naurp openstack-common-0.1+git20111130.orig/setup.py openstack-common-0.1+git20111130/setup.py
--- openstack-common-0.1+git20111130.orig/setup.py	2011-11-30 13:39:09.000000000 -0500
+++ openstack-common-0.1+git20111130/setup.py	2011-12-05 13:06:45.277528424 -0500
@@ -32,7 +32,6 @@ Common components for Openstack includin
           'routes',
           'webob',
           'nose',
-          'nose-exclude',
           'mox',
           'webtest',
           'lxml',
