--- a/openstack/common/setup.py
+++ b/openstack/common/setup.py
@@ -36,6 +36,11 @@
     return mapping
 
 
+def str_dict_replace(s, mapping):
+    for s1, s2 in mapping.iteritems():
+        s = s.replace(s1, s2)
+    return s
+
 def canonicalize_emails(changelog, mapping):
     """ Takes in a string and an email alias mapping and replaces all
         instances of the aliases in the string with their real email
